src/media/js/settings_app.js

```diff
@@ -76,6 +77,8 @@ define('settings', ['l10n', 'settings_local', 'underscore'], function(l10n, sett
         persona_tos: null,
         persona_privacy: null,
 
-        title_suffix: 'Commonplace App'
+        title_suffix: 'Operator Dashboard'
     });
+
+    settings._extend(settingsLocal);
```

Comment:
Looks good, the `settings_local` files shouldn't need to change because it's handled here.